File: DivinoLanches/ClientApp/src/models/VendaModel.ts
export class VendaModel {

    id: number
    nomeProduto: number
    tipoProduto: number
    subTipoProduto: number
    formaPagamento: string
    quantidade: number
    valorTotal: number
    dataVenda: string

    dataDe: string
    dataAte: string

    constructor(obj: any) {
        this.id = obj.id
        this.nomeProduto = obj.nomeProduto
        this.tipoProduto = obj.tipoProduto
        this.subTipoProduto = obj.subTipoProduto
        this.quantidade = obj.quantidade
        this.valorTotal = obj.valorTotal       
        this.formaPagamento = obj.formaPagamento       
        this.dataVenda = obj.dataVenda
        this.dataDe = obj.dataDe
        this.dataAte = obj.dataAte
        return this
    }
}

File: DivinoLanches/Models/VendaModel.cs
using System;

namespace DivinoLanches.Models
{
    public class VendaModel
    {
        public int Id { get; set; }
        public string NomeProduto { get; set; }
        public string TipoProduto { get; set; }
        public string SubTipoProduto { get; set; }
        public string FormaPagamento { get; set; }
        public int Quantidade { get; set; }
        public double ValorTotal { get; set; }
        public string DataVenda { get; set; }
        public string DataDe { get; set; }
        public string DataAte { get; set; }
    }
}
